### PR TITLE
Set 'unsubscribed' in Customer.io for new email users.

### PR DESCRIPTION
**Pull request by DFurnes**
### Changes
This pull request adds some logic to determine whether a user is newly created by comparing their `created_at` and `updated_at` values, which means we'll be able to start sending user updates without automatically re-subscribing people.

We're currently assuming that `unsubscribed` is only relevant for users with email addresses, since we want to track SMS subscription status separately in the `sms_status` field. (Waiting to hear back from Customer.io to see if we need to do some extra work here!)

### How should this be reviewed?
I updated the user returned from the `getValidUser` method to be "new" so that the existing tests continue to pass. If it makes sense, I'm also up for adding a few new test cases here for user updates & new users without email addresses.

### CODE REVIEW

**Comment by DFurnes** [comment]
Ran through some manual testing against #133 and #134 on my local, and it seems that the correct payloads are being sent to Customer.io for each of the different conditions. Would definitely appreciate a second set of eyes once this is deployed as well!

File: src/messages/CustomerIoUpdateCustomerMessage.js [hunk 1]
```diff
@@ -112,7 +112,10 @@ class CustomerIoUpdateCustomerMessage extends Message {
       ).unix();
     }
 
-    if (isNew) {
+    // If a user is newly created (created_at & updated_at are the same)
+    // then set them as "subscribed" to emails in Customer.io!
+    const isNew = customerData.created_at === customerData.updated_at;
+    if (customerData.email && isNew) {
```
Comment (sergii-tkachenko) [inline src/messages/CustomerIoUpdateCustomerMessage.js]: Hm. I'm in doubt if we need to track `subscribed_at` timestamp for new members with no email. Let's discuss it with Kika.

Comment (DFurnes) [reply src/messages/CustomerIoUpdateCustomerMessage.js]: Confirmed this is [no longer needed](https://dosomething.slack.com/archives/C2C8NLNAY/p1506964694000341). Removed in c44e0b9.

**Comment by sergii-tkachenko** [comment]
Tested on staging!

**Pull request merged**